### DISCUSSION TITLE
How to add Google fonts?

### DISCUSSION THREAD

**Discussion by alaireselene** [vuepress-theme-hope/vuepress-theme-hope#1578] [Q&A] [answered]
Hello, I've read the instruction and modified ".vuepress/styles/palette.scss"
But the font seem to be not changed.
Here is my code:
```scss
@import url('https://fonts.googleapis.com/css2?family=Fira+Sans&family=IBM+Plex+Mono&display=swap');
// font
$fontFamily: '-apple-system, BlinkMacSystemFont, "Segoe UI", "Fira Sans", sans-serif' !default;
$fontFamilyFancy: '-apple-system, BlinkMacSystemFont, "Segoe UI", "Fira Sans", sans-serif' !default;

// code
$fontFamilyCode: '"IBM Plex Mono", monospace' !default;
$lineNumbersWidth: 2.5rem !default;
```
So what is the right way to change fonts? Thank you so much Mr.Hope for your work!

**Comment by Mister-Hope** [comment, marked as answer]
Your problem is similar to this.

![6829E232-18F9-4794-8C62-F83C25C8BB5C](https://user-images.githubusercontent.com/33315834/158322467-dbe5340b-86fa-4ce0-aff0-0d3fb46a9fef.png)


https://vuepress-theme-hope.github.io/v2/faq.html#importing-iconfont-icons-does-not-work

**Reply by alaireselene** [reply]
Seem to be not worked for me:
```ts
head: [
    [
      "link",
      {
        rel: "preload",
        as: "style",
        onload: 'this.onload=null;this.rel="stylesheet"',
        href: "https://gist.githubusercontent.com/alaireselene/70cef1b86bdcc090868b64e5b029f3f1/raw/ae457d2d9a317b2c82be7eee69a040ec3367031c/style.css",
      },
    ],
  ],
```
And [Here](https://gist.githubusercontent.com/alaireselene/70cef1b86bdcc090868b64e5b029f3f1/raw/ae457d2d9a317b2c82be7eee69a040ec3367031c/style.css) is the css file which I embed to page.

**Reply by Mister-Hope** [reply]
Try using it the way docs are doing. I will update related docs.

**Reply by Mister-Hope** [reply]
Update, vuepress seems to have some issues creating tag with `onload` attribute. So the above line which lazy load the css is not working.

**Reply by alaireselene** [reply]
So up to now there are no solution sir? Maybe I must wait or return back to V1.

**Reply by Mister-Hope** [reply]
The docs has been updated

https://vuepress-theme-hope.github.io/v2/faq.html#importing-iconfont-icons-does-not-work

If you import like this it will work, but it will load sync instead of async
